packages/@okta/vuepress-site/guides/sign-into-spa/section03/angular/config.md

```diff
@@ -1,2 +1,29 @@
-In your application's `module.ts` file, import the following objects and create a configuration object:
-{code}
\ No newline at end of file
+Add OktaAuthModule module to the imports section of your application module's `@NgModule` declaration as shown below.
```

Comment:
Add the `OktaAuthModule` module to the imports section of your application module's `@NgModule` declaration.